spec/compiler/semantic/instance_var_spec.cr

```diff
@@ -4429,6 +4429,21 @@ describe "Semantic: instance var" do
       ), inject_primitives: false) { types["Foo"] }
   end
 
+  it "is more permissive with macro def initialize, bug with named args" do
```

Comment:
Reference this PR number in the spec name?

Reply:
That's a bit recursive :-)

I usually reference the issue number (bug), but there's none here (well, there's that comment...). In any case, the blame of that line will lead to this commit, and in GitHub it will relate to this PR.

Reply:
git blame can easilly be thrown off by a few refactors or file renames, far down the line.